app/views/layouts/_user_input_filter.html.haml

```diff
@@ -23,19 +23,25 @@
                                    options_for_select([_("<Choose>")] + MiqExpression.get_entry_details(@edit[:qs_tokens][token][:tag]).sort_by { |desc, _name| desc.downcase },
                                                        nil),
                                    :multiple              => false,
-                                   :class                 => "widthed",
+                                   :class    => "selectpicker",
```

Comment:
The hash rockets should be aligned with the rest (same for other changes like this in this PR).